### PR TITLE
Fix updateReadme command and Remove duplicate 'Googlebot' from README

### PR DESCRIPTION
**Pull request by shane-smith**
### Description:

Googlebot appears twice in the list of detected bots, which is clearly a mistake.

This simple change removes one of them.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by shane-smith** [comment]
It looks like `WooRank` also appears twice. I haven't fixed that in this branch, but a maintainer could do that before merging?

**Comment by sanchezzzhak** [comment]
Hi, update the readme happens automatically via the command  `php misc/updateReadme.php`

I think it's worth solving the problem there https://github.com/matomo-org/device-detector/blob/cae491e21ce05d254b71467a7c4a4364805ada59/misc/updateReadme.php#L37C1-L37C20
```php
natcasesort(array_unique($bots)); 
```

**Comment by shane-smith** [comment]
Okay! 😁

I'm happy to make that change and resubmit the PR.

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**
